src/sounds.cpp

```diff
@@ -204,12 +209,21 @@ void sounds::process_sound_markers( player *p )
             continue;
         }
 
+        // At this point we are dealing with attention (as opposed to physical effects)
+        // so reduce volume by the amount of ambient noise from the weather.
+        const int mod_vol = volume - weather_vol;
+
+        // The noise was drowned out by the surroundings
+        if (mod_vol <= 0) {
```

Comment:
Can you please change this to `< 0`, there are some sounds in iuse.cpp (bombs ticking) that use volume=0 and they should be heard in silent weather. Or add another check to skip the sound if it's 0 *and* weather_vol is not 0. With the current comparison, the bomb ticking is never heard.